tools/clearShm/main.cpp

```diff
@@ -79,6 +102,29 @@ void shmDoit(key_t shm_key, const string& label)
     }
 }
 
+void semDoit(std::string segmentName, const string& label)
```

Comment:
What is the diff b/w shmDoit and semDoit? They looks almost identical and it is better to reduce boilerplate code.

Reply:
They are different, the original one has a input parameter as a key, transforms the key to name, in my implementation the key is not a name, the name is fixed.

Reply:
UPD, I'fixed that, thanks.